packages/nextra/src/compile.ts

```diff
@@ -61,11 +63,13 @@ export async function compileMdx(
     | 'defaultShowCopyCode'
     | 'readingTime'
     | 'latex'
+    | 'codeHighlight'
   > & {
     mdxOptions?: LoaderOptions['mdxOptions'] &
-      Pick<ProcessorOptions, 'jsx' | 'outputFormat' | 'format'>
+      Pick<ProcessorOptions, 'jsx' | 'outputFormat'>
```

Comment:
I removed `format` from here because we already specified it in `mdxOptions.format` and there are conflicts